### PR TITLE
Add file permission notes to readme

### PR DESCRIPTION
**Pull request by Jhoyola**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,8 @@ lightningd --plugin=/path/to/plugin/directory/plugin_file_name.py
 Notes:
  - The `plugin_file_name.py` must have executable permissions:
    `chmod a+x plugin_file_name.py`
+   - You must have git core.fileMode set to true to reflect the permissons in git
```
Comment (chrisguida) [inline README.md:72]:
```suggestion
   - You must have git core.fileMode set to true to reflect the permissions in git
```

**Review by chrisguida — CHANGES_REQUESTED** [review]
see misspelling

**Comment by chrisguida** [comment]
Please squash

**Comment by Jhoyola** [comment]
I'll squash tomorrow 👍

**Review by chrisguida — APPROVED** [review]
ACK https://github.com/lightningd/plugins/commit/52400a86b17b2cc87f3c094285daf6b6119de335

**Comment by chrisguida** [comment]
I don't know why but this PR seems to be stuck "processing updates"

image

Turned on automerge to try to see if it will merge itself once it gets unstuck

**Comment by chrisguida** [comment]
Yep haha

I guess just some random bug in GH

re-ACK https://github.com/lightningd/plugins/pull/579/commits/d47003aa8ae308931d5895d36efd6fa3f7562456

**Pull request merged**